phaseshifts/phsh.py

```
@@ -662,7 +663,7 @@ def main(argv=None):
             "--tmpdir",
             dest="tmpdir",
             metavar="<temp_dir>",
-            help="temporary directory for intermediate file generation",
+            help="temporary directory for intermediate " "file generation",
```

Ruff (0.14.10)

Implicitly concatenated string literals on one line

Combine string literals

(ISC001)